modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/build.gradle.kts.mustache

```diff
@@ -0,0 +1,119 @@
+plugins {
+    kotlin("multiplatform") version "1.3.72"
+    kotlin("plugin.serialization") version "1.3.72"
+}
+
+group = "{{groupId}}"
+version = "{{artifactVersion}}"
+
+val kotlin_version = "1.3.72"
+val ktor_version = "1.3.2"
+
+repositories {
+    jcenter()
+}
+
+kotlin {
+    /*
+     * To find out how to configure the targets, please follow the link:
+     * https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#setting-up-targets
+     */
+    jvm {
+        val main by compilations.getting {
+            kotlinOptions {
+                // Setup the Kotlin compiler options for the 'main' compilation:
+                jvmTarget = "1.8"
+            }
+        }
+    }
+    ios {
+        binaries {
+            framework {
+                freeCompilerArgs = listOf("-Xobjc-generics")
+            }
+        }
+    }
```

Comment:
I can't test IOS

Reply:
So I basically need someone with a mac who can write a quick unittest for pet store or so and then run the test